crates/web-sys/src/features/gen_HtmlHrElement.rs

```diff
@@ -0,0 +1,39 @@
+use super::*;
+use wasm_bindgen::prelude::*;
+#[wasm_bindgen]
+extern "C" {
+    # [ wasm_bindgen ( extends = HtmlElement , extends = Element , extends = Node , extends = EventTarget , extends = :: js_sys :: Object , js_name = HTMLHRElement , typescript_name = HTMLHRElement ) ]
+    #[derive(Debug, Clone, PartialEq, Eq)]
+    #[doc = "The `HtmlHrElement` class.\n\n[MDN Documentation](https://developer.mozilla.org/en-US/docs/Web/API/HTMLHRElement)\n\n*This API requires the following crate features to be activated: `HtmlHrElement`*"]
```

Comment:
So a random comment about the style of these files. Unfortunately I don't think there's anything we can do about `# [ wasm_bindgen ]` so we can just sort of live with reading that. For `#[doc]` though one thing we could improve on is to split the doc string on newlines and generate a `#[doc]` annotation per line, which I think might format a bit more nicely? Just a thought though.

Reply:
Yeah, I thought about that, I think that's a good idea.

Reply:
It now generates one `#[doc]` per line.